client/assets/pages/_events-page.scss

```diff
@@ -540,3 +540,14 @@ main {
   text-transform: capitalize;
   margin-bottom: 5px;
```

Comment:
This might not be in your scope but where did this *px* come from 😕

Reply:
I created a [chore](https://www.pivotaltracker.com/n/projects/2174978/stories/160507917) to refactor our css since we've been lying behind on some our best practices